docs/00-manuscripts/ieee/reviews/reviewer3.md

```diff
@@ -22,26 +22,22 @@ Thanks for pointing towards the information redundancy characteristic of the dat
 Feature sets NRI and VI are more diverse and require four (NRI) and five (VI) PC to explain 90% of the variance in the data.
 
 > 3. RF is usually effected by the randomization of the training process, did you only run one shot, or it is the average result from multiple trails?
 
 As outlined in "Methods - Benchmarking Design - Spatial Resampling" a fixed spatial CV on the fold level was conducted.
 This means that each model was fitted four times with different training and test sets.
-With respect to RF, 500 trees (hyperparameter `n_trees`) were fitted in each model fit.
-This hyperparameter value is not listed in the table of used hyperparameters because it was kept static (with 500 trees being the default of RF) and not tuned.
+With respect to RF, 500 trees (hyperparameter `n_trees`) were fitted in each model fit, which practically eliminates any internal random variability between different repetitions on the same training set.
 
 Because we used a partioning method which relies on fixed data splits (1 fold = 1 plot), no more than four runs were possible.
 
 > 4. The main novelty should be emphasized again which to me it is not significant and clear at this stage of presentation. Some conclusions are commonly known to all for hyperspectral feature extraction and classification. So which are the unique contributions or novelties from this work should be highlighted again.
 
 Thanks for pointing this out.
-Apart from showcasing best practices of hyperspectral data processing and modelling, so far no work was done using filter methods (and especially ensemble filters) in a high-dimensional feature space retrieved from hyperspectral remote sensing data.
+So far no work has been published using and comparing filter methods (and especially ensemble filters) in a high-dimensional feature space retrieved from hyperspectral remote sensing data. Ensemble filters as an innovative approach that combines multiple filter techniques is even less known in remote-sensing and environmental modelling.
 The special case here is that hyperspectral-based high-dimensional data is different to high-dimensional data from non-hyperspectral datasets due to the high similarity of the input layers.
-In our view, the use of filters methods in machine learning is pretty much non-existent yet, especially in environmental modeling.
-This study shows how these can possibly be used in high-dimensional scenarios on a real world dataset.
-In addition, it compares the performance of ensemble filters in such a scenario.
-Last, the complete study is embedded in a single ML framework with the full code being provided to make the study fully reproducible and possibly reusable in future studies.
+In addition, the complete study is embedded in a single ML framework with the full code being provided to make the study fully reproducible and possibly reusable in future studies. We anticipate that this will enhance the impact of our work.
```

Comment:
```suggestion
In addition, the complete study is embedded in a single ML framework with the full code being provided to make the study fully reproducible and possibly reusable in future studies.
We anticipate that this will enhance the impact of our work.
```